<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::WebsitesPreferencesPage</class>
 <widget class="QWidget" name="Otter::WebsitesPreferencesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="websitesContentLayout">
     <item>
      <widget class="Otter::LineEditWidget" name="websitesFilterLineEditWidget">
       <property name="placeholderText">
        <string>Search…</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Otter::ItemViewWidget" name="websitesItemView">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>250</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="websitesButtonsLayout" stretch="0,0,0,1">
     <item>
      <widget class="QPushButton" name="websitesAddButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="websitesEditButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit…</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="websitesRemoveButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="websitesButtonsSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::ItemViewWidget</class>
   <extends>QTreeView</extends>
   <header>src/ui/ItemViewWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Otter::LineEditWidget</class>
   <extends>QLineEdit</extends>
   <header>src/ui/LineEditWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
